rotkehlchen/db/filtering.py

```diff
@@ -673,6 +698,7 @@ def make(
             ignored_ids: Optional[List[str]] = None,
             null_columns: Optional[List[str]] = None,
             event_identifier: Optional[str] = None,
+            counterparty: Optional[str] = None,
```

Comment:
I would make this both here and in the given api a list of counterparties. There is no need to filter only by one protocol. User should be able to filter by multiple protocols.